cmd/cleanup/main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"time"
+
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/sig-storage-local-static-provisioner/pkg/cleanup/controller"
+	"sigs.k8s.io/sig-storage-local-static-provisioner/pkg/cleanup/deleter"
+)
+
+// Command line flags
+var (
+	kubeconfig               = flag.String("kubeconfig", "", "Absolute path to the kubeconfig file. Only required when running out of cluster.")
+	masterURL                = flag.String("master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
+	resync                   = flag.Duration("resync", 10*time.Minute, "Resync interval of the controller.")
+	storageClassName         = flag.String("storageClassName", "", "Name of StorageClass to opt-in PVs and PVCs for cleanup.")
+	workerThreads            = flag.Uint("worker-threads", 10, "Number of controller worker threads.")
+	delay                    = flag.Duration("delay", 1*time.Minute, "How much time to wait after Node deletion for resource cleanup.")
```

Comment:
I would name the parameter more specific, like "pvc-deletion-delay". Also in the help text, specify the units.

Reply:
Updated and specified seconds as the unit, though I think that any passed in unit should work given that the flag.Duration is parsed by time.ParseDuration().